setup.py

```diff
@@ -1,12 +1,28 @@
 from setuptools import setup
 
-setup(name='tobac',
-      version='1.2',
-      description='Tracking and object-based analysis of clouds',
-      url='http://github.com/climate-processes/tobac',
-      author='Max Heikenfeld',
-      author_email='max.heikenfeld@physics.ox.ac.uk',
-      license='GNU',
-      packages=['tobac'],
-      install_requires=[],
-      zip_safe=False)
+setup(
+    name="tobac",
+    version="1.3",
+    description="Tracking and object-based analysis of clouds",
+    url="http://github.com/climate-processes/tobac",
+    author=[
+        "Max Heikenfeld",
+        "William Jones",
+        "Fabian Senf",
+        "Sean Freeman",
+        "Julia Kukulies",
+        "Peter Marinescu",
+    ],
+    author_email=[
+        "max.heikenfeld@physics.ox.ac.uk",
+        "william.jones@physics.ox.ac.uk",
+        "senf@tropos.de",
+        "sean.freeman@colostate.edu",
+        "julia.kukulies@gu.se",
+        "peter.marinescu@colostate.edu",
+    ],
+    license="GNU",
```

Comment:
I'm not sure if this is worth stopping the release over, but this is not correct in `main` either. Our license per the `LICENSE` file is the BSD 3-clause license.

Reply:
Good catch, I'll fix that quickly